linux-package.sh

```diff
@@ -16,15 +16,14 @@ javapackager -deploy \
     -BappVersion=$ver \
     -Bcategory=Office,Finance \
     -BlicenseType=Apache \
+    -BlicenseFile=LICENSE \
     -Bemail=contact@vinumeris.com \
     -Bicon=client/icons/icon.png \
     -native deb \
     -name lighthouse \
-    -title Lighthouse \
+    -title "Lighthouse: a peer to peer crowdfunding app that uses Bitcoin" \
     -vendor Vinumeris \
     -outdir deploy \
     -appclass lighthouse.Main \
-    -srcfiles $build \
+    -srcdir updates/builds/processed \
```

Comment:
No, only one JAR will be packaged, along with license file and postinst / prerm scripts. The directory contents are erased before build in package.sh.

An alternative solution is to keep old updates in the processed folder, and use -srcfiles to specify only $build, postinst, prerm and LICENSE. Would you prefer this solution? I can add another commit, it's a small change.